nmdc_server/schemas.py

```diff
@@ -632,3 +632,8 @@ class Config:
 
 class KeggTermTextListResponse(BaseModel):
     terms: List[KeggTermText]
+
+
+class IngestArgumentSchema(BaseModel):
+    skip_annotation: bool = False
+    function_limit: int = 0  # Default to no limit
```

Comment:
Is passing `0` the same as passing `None`?

Reply:
They result in the same behavior, if that's what you mean.

`Optiona[int]` would hide the option from the generated example JSON, which makes it pretty much undiscoverable